### DISCUSSION TITLE
GitHub Octernship query

### DISCUSSION THREAD

**Discussion by AbhishekRP2002** [community/community#49750] [GitHub Education]
Can anyone please give some overview for how we can submit the task assigned for GitHub Octernship ?

**Comment by msuliq** [comment]
Hi @AbhishekRP2002,

Here are some instructions from [Git and GitHub Classroom tutorial](https://lse-me314.github.io/assignment01/lab-session/GIT.html)
Handing in your homework amounts to committing changes you have made in your repository. There are 3 ways to do it:
**1. Use RStudio**
  - Select the “Git” tab and check all the files that are changed. Hit the “Commit” button.
  - In the window that open, write a commit message. Then push the “Commit” button.
  - To push the changes to GitHub, simply click on “Push” and enter the required credentials.
**2. Use the command line**
```
cd [your github repository location]
git add "*"
git commit -m "your comment"
git push origin master
```
[Reference here](https://stackoverflow.com/questions/10364429/how-to-commit-to-remote-git-repository )

**3. Update files on Github.com:**
You can upload your updated files by clicking the “upload files” button ([see here](https://help.github.com/articles/adding-a-file-to-a-repository/))  to replace the outdated files (NOT recommended, you should be able to use the commit and push!)
if your files are in some simple format (like .md), you can edit the files directly

It is always a good idea to check if your commits are reflected on Github. If this helped, please mark this comment as accepted answer. Thank you
